docs/faq.md

```diff
@@ -355,3 +356,15 @@ Run the following commands:
 containerd-rootless-setuptool.sh uninstall
 rootlesskit rm -rf ~/.local/share/containerd ~/.local/share/nerdctl ~/.config/containerd
 ```
+
+### How to clean a dangling cache of buildkit?
+
+`buildkit` cache directory is located at `$HOME/.local/share/bulidkit/`
+in rootless mode, which has same folder structure `/var/lib/buildkit/` in
+root mode.
+
+You can clear the cache objects by running the following command:
+```
+nerdctl builder prune
```

Comment:
Out of curiosity; why is nerdctl system prune --all? not enough

Reply:
It is significant effective without `--all`. It is not clear what `--all` includes.